recipes/arraylake/meta.yaml

```
@@ -0,0 +1,78 @@
+{% set name = "arraylake" %}
+{% set version = "0.7.2a1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/arraylake-{{ version }}.tar.gz
+  sha256: d5674aac64c3aac2c02b0ed38d250c0c9d30c37c8428f359d39cb2e9269370a5
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  entry_points:
+    - arraylake = arraylake.cli.main:app
+    - al = arraylake.cli.main:app
+requirements:
```

Newline from the GH suggestion was lost

```suggestion
    - al = arraylake.cli.main:app

requirements:
```